sdcm/fill_db_data.py

```diff
@@ -3387,7 +3388,7 @@ def fill_table():
                 assert f'Query "{statement}" returned no entries'
             self.log.info('running now fetch_all_rows')
             full_res = self.rows_to_list(
-                self.fetch_all_rows(session=session, default_fetch_size=100, statement=statement))
+                fetch_all_rows(session=session, default_fetch_size=100, statement=statement))
```

Comment:
Did you test the rolling upgrade with this change ?

Reply:
Tested in https://jenkins.scylladb.com/job/scylla-staging/job/yarongilor/job/rolling-upgrade-ubuntu20.04-test-yg/2/

Reply:
it failed with spot termination... @yarongilor , please confirm if this part of the code was called, and passed correctly

Reply:
@fgelcer , @fruch , this code passed ok:
```
< t:2023-10-16 12:03:36,505 f:database_query_utils.py l:209  c:sdcm.utils.database_query_utils p:DEBUG > Fetch all rows by statement: select * from keyspace_complex.paged_query_test;
< t:2023-10-16 12:03:37,611 f:database_query_utils.py l:223  c:sdcm.utils.database_query_utils p:DEBUG > Size of fetched rows: 84000 bytes
```